test/fuzz/cne_log.cc

```diff
@@ -0,0 +1,33 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2021-2022 Intel Corporation
+ */
+
+#include <cne_log.h>
+
+/* Prototype required to fix "no previous prototype for function" error */
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size);
+
+extern "C" int
+LLVMFuzzerTestOneInput(const uint8_t *data_, size_t size)
+{
+    char *data;
+
+    /* Need at least one byte for null terminator */
+    if (size < 1)
+        return 0;
+    /* Create null-terminated string from input data */
+    data = (char *)calloc(size + 1, sizeof(char));
+    if (!data)
+        return 0;
+    strncpy(data, (const char *)data_, size);
+
+    /* Replacing '%' with another character. This is needed to
+     *  avoid SEGV on unknown address in Address Sanitizer. */
```

Comment:
Now there's an extra space :(. Oh well, we'll fix it next time this file needs to be changed.